src/resolver/package_json.zig

```diff
@@ -667,6 +669,22 @@ pub const PackageJSON = struct {
             }
         }
 
+        if (json.asProperty("config")) |npm_pkg_cfg| {
+            switch (npm_pkg_cfg.expr.data) {
+                .e_object => |obj| {
+                    for (obj.properties.slice()) |*prop| {
+                        const key = prop.key.?.asString(allocator) orelse continue;
+                        const value = prop.value.?.asString(allocator) orelse continue;
```

Comment:
I'm pretty sure it's supposed to work with arbitrary objects, like `config: { foo: { bar: 1 } }` becomes `$npm_config_foo_bar=1`

Reply:
That being said, it'd probably be fine to skip objects for the first version of this since copying them at all would strictly be an improvement over status quo